website/docs/execution-node/fee-recipient.md

```diff
@@ -291,6 +203,24 @@ See [this issue](https://github.com/prysmaticlabs/prysm/issues/11322) to track t
 
 :::
 
+
+## Frequently asked questions
+
+**How do I know if fee recipient was properly configured?** <br />
+If you don't see any errors after issuing one of the above commands, your fee recipient address has been successfully configured.
```

Comment:
I am trying to improve this with logs at the start of the validator client and beacon node see [pr](https://github.com/prysmaticlabs/prysm/pull/11307)

Reply:
this pr is also trying to fix the persistence bug,